show/chassis_modules.py

```diff
@@ -102,3 +108,55 @@ def midplane_status(chassis_module_name):
         click.echo(tabulate(table, header, tablefmt='simple', stralign='right'))
     else:
         click.echo('No data available in CHASSIS_MIDPLANE_TABLE\n')
+
+@chassis.command()
+@click.argument('systemportname', required=False)
+@click.option('--namespace', '-n', 'namespace', required=True if multi_asic.is_multi_asic() else False, 
+                default=None, type=str, show_default=False, help='Namespace name or all')
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def system_ports(systemportname, namespace, verbose):
+    """Show VOQ system ports information"""
+
+    cmd = "voqutil -c system_ports"
+
+    if systemportname is not None:
+        cmd += " -i \"{}\"".format(systemportname)
+    
+    if namespace is not None:
+        cmd += " -n {}".format(namespace)
+
+    clicommon.run_command(cmd, display_cmd=verbose)
+
+@chassis.command()
+@click.argument('ipaddress', required=False)
+@click.option('--asicname', '-x', 'asicname', default=None, type=str, show_default=False, help='Asic name')
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def system_neighbors(asicname, ipaddress, verbose):
+    """Show VOQ system neighbors information"""
+
+    cmd = "voqutil -c system_neighbors"
+
+    if ipaddress is not None:
+        cmd += " -a {}".format(ipaddress)
+
+    if asicname is not None:
+        cmd += " -x {}".format(asicname)
+
+    clicommon.run_command(cmd, display_cmd=verbose)
+
+@chassis.command()
+@click.argument('systemlagname', required=False)
+@click.option('--asicname', '-x', 'asicname', default=None, type=str, show_default=False, help='Asic name')
```

Comment:
why not use `-n` here ? similar to `show chassis system_ports`

Reply:
For system neighbors and system lags the info are retrieved from supervisor card's chassis app db which is not in any particular namespace. The 'asicname' here is not the namespace. It is the 'asic_name' we look for in the keys of the entries.

Reply:
Changed "asicname" option selector to "-n" as suggested here and in sonic chassis subgroup review meeting held on 08/04/2021